vtex-apps/digital-river
# [DIGITALRIV-30] Ignore capitalization for countryOfOrigin and eccn specifications

#### What problem is this solving?

If the product specifications `Country of origin` and `ECCN` were setup in the Catalog differently (e.g. `Country of Origin`, `eccn`, etc.) then the SKU would return an error when syncing the Catalog with Digital River. This PR ignores casing when looking for these specifications.

#### How to test it?
Test in workspace [annadigitalriver](https://annadigitalriver--sandboxusdev.myvtex.com/admin/digital-river/catalog-logs) in account sandboxusdev.

I changed the product specification from `Country of origin` to `Country of Origin`. The broadcaster will send the SKU change notification to both the annadigitalriver and master workspaces. Try updating the value for the `Country of Origin` for product ID [880003](https://sandboxusdev.myvtex.com/admin/Site/ProdutoForm.aspx?id=880003). After it reindexes, notice that the SKU is sent once without `countryOfOrigin` and once with.

#### Screenshots or example usage:
_Product specifications_
<img width="758" alt="Screen Shot 2022-08-24 at 1 39 03 PM" src="https://user-images.githubusercontent.com/53097865/186486674-551d3727-80b5-4dfa-b161-cae32e82fcb8.png">

_Country of origin sent in annadigitalriver (in blue). Country of origin not sent in master workspace (in red)._
<img width="1437" alt="Screen Shot 2022-08-24 at 11 51 42 AM" src="https://user-images.githubusercontent.com/53097865/186485856-99fbb0bb-63e2-479f-9bb1-a144ac2050c1.png">
